bench/src/bench.rs

```diff
@@ -133,37 +153,52 @@ fn instant_to_civil_datetime_static(c: &mut Criterion) {
 
     #[cfg(unix)]
     {
-        if let Ok(tz) = tzfile::Tz::named(TZNAME) {
+        if let Ok(tz) = tzfile::Tz::named(tz_name) {
             let expected = chrono::NaiveDateTime::new(
                 chrono::NaiveDate::from_ymd_opt(2024, 6, 30).unwrap(),
                 chrono::NaiveTime::from_hms_opt(9, 46, 0).unwrap(),
             );
-            c.bench_function(&format!("chrono-tzfile/{NAME}"), |b| {
-                use chrono::TimeZone;
-
-                b.iter(|| {
-                    let mapped = (&tz).timestamp_opt(bb(STAMP), 0);
-                    let zdt = mapped.single().unwrap();
-                    assert_eq!(zdt.naive_local(), expected);
-                })
-            });
+            c.bench_function(
+                &format!("chrono-tzfile/{NAME}-{tz_name}"),
+                |b| {
+                    use chrono::TimeZone;
+
+                    b.iter(|| {
+                        let mapped = (&tz).timestamp_opt(bb(timestamp), 0);
+                        let zdt = mapped.single().unwrap();
+                        assert_eq!(zdt.naive_local(), expected);
+                    })
+                },
+            );
         }
     }
 
-    let expected = chrono::NaiveDateTime::new(
-        chrono::NaiveDate::from_ymd_opt(2024, 6, 30).unwrap(),
-        chrono::NaiveTime::from_hms_opt(9, 46, 0).unwrap(),
-    );
-    c.bench_function(&format!("chrono/{NAME}"), |b| {
-        use chrono::TimeZone;
-        use chrono_tz::America::New_York;
+    use chrono_tz::America::New_York;
+    use chrono_tz::Asia::Shanghai;
+    use chrono_tz::Tz;
```

Comment:
Please follow the style of the surrounding code. Move the `use` statements to the top of the function and combine them into one.